tap_github/streams.py

```diff
@@ -0,0 +1,753 @@
+    replication_method = "INCREMENTAL"
+    replication_keys = "updated_at"
+    key_properties = ["id"]
+    path = "pulls?state=all"
+    children = ['reviews', 'review_comments', 'pr_commits']
+    pk_child_fields = ["number"]
+
+class ProjectCards(IncrementalStream):
+    '''
+    https://docs.github.com/en/rest/reference/projects#list-project-cards
+    '''
+    tap_stream_id = "project_cards"
+    replication_method = "INCREMENTAL"
+    replication_keys = "updated_at"
+    key_properties = ["id"]
+    path = "projects/columns/{}/cards"
+    tap_stream_id = "project_cards"
+    parent = 'project_columns'
+    id_keys = ['id']
+
+class ProjectColumns(IncrementalStream):
+    '''
+    https://docs.github.com/en/rest/reference/projects#list-project-columns
+    '''
+    tap_stream_id = "project_columns"
+    replication_method = "INCREMENTAL"
+    replication_keys = "updated_at"
+    key_properties = ["id"]
+    path = "projects/{}/columns"
+    children = ["project_cards"]
+    parent = "projects"
+    id_keys = ['id']
+    has_children = True
+
+class Projects(IncrementalStream):
+    '''
+    https://docs.github.com/en/rest/reference/projects#list-repository-projects
+    '''
+    tap_stream_id = "projects"
+    replication_method = "INCREMENTAL"
+    replication_keys = "updated_at"
+    key_properties = ["id"]
+    path = "projects?state=all"
+    tap_stream_id = "projects"
+    children = ["project_columns"]
+    child_objects = [ProjectColumns()]
+
+class TeamMemberships(FullTableStream):
+    '''
+    https://docs.github.com/en/rest/reference/teams#get-team-membership-for-a-user
+    '''
+    tap_stream_id = "team_memberships"
+    replication_method = "FULL_TABLE"
+    key_properties = ["url"]
+    path = "orgs/{}/teams/{}/memberships/{}"
+    is_organization = True
+    parent = 'team_members'
+    id_keys = ["login"]
+
+    def add_fields_at_1st_level(self, rec, parent_record):
+        """
+        Add fields in the record explicitly at the 1st level of JSON.
+        """
+        rec['login'] = parent_record['login']
+
+class TeamMembers(FullTableStream):
+    '''
+    https://docs.github.com/en/rest/reference/teams#list-team-members
+    '''
+    tap_stream_id = "team_members"
+    replication_method = "FULL_TABLE"
+    key_properties = ["team_slug", "id"]
+    path = "orgs/{}/teams/{}/members"
+    is_organization = True
+    id_keys = ['slug']
+    children= ["team_memberships"]
+    has_children = True
+    parent = 'teams'
+    pk_child_fields = ['login']
+
+
+    def add_fields_at_1st_level(self, rec, parent_record):
+        """
+        Add fields in the record explicitly at the 1st level of JSON.
+        """
+        rec['team_slug'] = parent_record['slug']
+
+class Teams(FullTableStream):
+    '''
+    https://docs.github.com/en/rest/reference/teams#list-teams
+    '''
+    tap_stream_id = "teams"
+    replication_method = "FULL_TABLE"
+    key_properties = ["id"]
+    path = "orgs/{}/teams"
+    is_organization = True
```

Comment:
Can we use a better name? as teams are not an organization. 
Suggestion: use_organization_only or use_organization

Reply:
Same for repo as well

Reply:
Updated `is_organization `ann `is_repository `to `use_organization_only `and `use_organization `respectively.